networkdb/cluster.go

```diff
@@ -107,7 +107,6 @@ func (nDB *NetworkDB) clusterInit() error {
 		config.BindPort = nDB.config.BindPort
 	}
 
-	config.ProtocolVersion = memberlist.ProtocolVersionMax
```

Comment:
After the memberlist bump in 17.06 there are still going to swarm clusters with nodes running versions < 17.05. So if memberlist code changes the default version to anything > 3 it will break us. I don't see any reason why we should take that chance.